src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.props

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
(for everyone else's info) We discussed offline in email.  This buys us the ability to use these values to build DependencyModel in core-setup, but still use the .NET Core SDK logic to generate the assembly info and the .nuspec/.nupkg files.